### ISSUE TITLE
Launch messaging

### ISSUE THREAD

**Issue by feinstruktur** [qutheory/vapor#326]
When running the project from Xcode the console just says
```
[1464327605] [INFO] Server starting …
```
whereas `vapor run` will print
```
Running...
Visit http://localhost:8080
[1464328127] [INFO] Server starting ...
[1464328133] [INFO] GET /
[1464328133] [INFO] GET /styles/app.css
[1464328133] [INFO] GET /images/vapor-logo.png
```

Should roll the that messaging into the same place (the first line `Runnin…` is from `vapor-cli`, not sure where `Visit…` is from yet) and make it clearer when the server is actually done starting up.

**Comment by tannernelson** [comment]
I think you may be running different projects. The "Visit" message comes from the Example project.

**Comment by tannernelson** [comment]
This was addressed in 0.11.
